src/views/shared/SecurityBeacon.js

```diff
@@ -120,7 +140,8 @@ define(['okta', 'util/Animations'], function (Okta, Animations) {
         <div class="circle right"></div>\
       </div>\
     </div>\
-    <div class="bg-helper auth-beacon auth-beacon-security" data-se="security-beacon">\
+    <div aria-live="polite" role="alert" class="bg-helper auth-beacon auth-beacon-security" data-se="security-beacon">\
```

Comment:
yeah, I don't know what is the best here either... maybe we should ask an expert if we can find one...

From [w3/roles](https://www.w3.org/TR/wai-aria/roles#alert)
> A message with important, and usually time-sensitive, information. 
> Alerts are used to convey messages to alert the user. In the case of audio warnings this is an accessible alternative for a hearing-impaired user.